static/locales/en-US.yaml

```diff
@@ -406,11 +406,16 @@ Settings:
   Player Settings:
     Player Settings: Player
     Play Next Video: Play Next Video
-    Turn on Subtitles by Default: Turn on Subtitles by Default
+    Turn on Subtitles by Default: Enable Subtitles by Default
     Autoplay Videos: Autoplay Videos
     Proxy Videos Through Invidious: Proxy Videos Through Invidious
     Autoplay Playlists: Autoplay Playlists
-    Enable Theatre Mode by Default: Enable Theatre Mode by Default
```

Comment:
This should be deleted for all locales, you might want to wait until everyone is ready to merge it before you do so to avoid code conflicts.